lib/CallBackery/GuiPlugin/AbstractCardlist.pm

```diff
@@ -0,0 +1,182 @@
+package CallBackery::GuiPlugin::AbstractCardlist;
+use Carp qw(carp croak);
+use CallBackery::Translate qw(trm);
+use CallBackery::Exception qw(mkerror);
+use Text::CSV;
+use Excel::Writer::XLSX;
```

Comment:
since this is not implemented anyway ... I would not load it

Reply:
Removed `CSV` and `XLSX`